addons/hr_expense/static/tests/mobile/expense_mobile.test.js

```diff
@@ -0,0 +1,26 @@
+import { describe, expect, test } from "@odoo/hoot";
+import { defineMailModels, openListView, registerArchs, start } from "@mail/../tests/mail_test_helpers";
+import { defineHrExpenseModels } from "../mock_server/mock_models/defineModel";
+import { tick } from "@odoo/hoot-mock";
+
+describe.current.tags("mobile");
+defineMailModels();
+defineHrExpenseModels();
```

Comment:
Couldn't we just call `defineHrExepenseModels` which would call `defineMailModels` ? 1 less import^^